src/EventPayload.ts

```diff
@@ -109,22 +147,21 @@ export interface EventPayload {
   searchTerm?: string;
   /** Unique identifier to tie together events in a single browsing session */
   sessionId?: string | null;
-  /** Fields specific to reporting Yext Pages Analytics Events */
-  pages?: {
-    /** The scope of an action. Applies to Pages events. */
-    scope?: string;
-    /* The UID of the site an event was tied to. */
-    siteUid?: number;
-    /* The ID of the template from which a site was generated. */
-    template?: string;
-  };
-  /** The timestamp at which the event occurred, in ISO format. */
+  /** The timestamp at which the event occurred, in ISO format.
+   * For example, September 27, 2022 at 6 p.m. is represented as
+   * 2022-09-27 18:00:00.000.
```

Comment:
I forget. Do we convert to UTC or is what is passed in presumed to be in UTC? If it is the latter, we probably should mention that. If not, this is fine.

Reply:
Discussed here https://yext.slack.com/archives/G01SSAYJVM4/p1709063238616539